<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Clicker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GCRes.qrc">
    <normaloff>:/Icons/images/main.png</normaloff>:/Icons/images/main.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="toolTip">
       <string>Select Script to start. Double click to edit it.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart_Script"/>
   <addaction name="actionStop_Script"/>
   <addaction name="actionEnable_Remote"/>
   <addaction name="separator"/>
   <addaction name="actionReload"/>
   <addaction name="actionSet_Script_s_Folder"/>
   <addaction name="actionWeb_Site"/>
   <addaction name="actionCheck_Updates"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>424</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Windows List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWindowTargeter" name="wSelector" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>220</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="title">
        <string>Add by Title</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>16</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Type window title to add to list.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>98</x>
          <y>14</y>
          <width>20</width>
          <height>23</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Click to add windows by typed title. Each window is added once.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>+1</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="toolTip">
        <string>Windows list. Double click to bring to front one.</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>27</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>Clear the windows' list.</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDummy"/>
  </widget>
  <action name="actionStart_Script">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="GCRes.qrc">
     <normaloff>:/Icons/images/Launch.png</normaloff>:/Icons/images/Launch.png</iconset>
   </property>
   <property name="text">
    <string>Start Script</string>
   </property>
   <property name="toolTip">
    <string>Starts or Re-starts script.</string>
   </property>
  </action>
  <action name="actionStop_Script">
   <property name="icon">
    <iconset resource="GCRes.qrc">
     <normaloff>:/Icons/images/Stop.png</normaloff>:/Icons/images/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Script</string>
   </property>
   <property name="toolTip">
    <string>Immedial stop and unregister hotkeys.</string>
   </property>
  </action>
  <action name="actionDummy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>dummy</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="GCRes.qrc">
     <normaloff>:/Icons/images/Reload all.png</normaloff>:/Icons/images/Reload all.png</iconset>
   </property>
   <property name="text">
    <string>Reload List</string>
   </property>
   <property name="toolTip">
    <string>Reload scripts' list</string>
   </property>
  </action>
  <action name="actionCheck_Updates">
   <property name="icon">
    <iconset resource="GCRes.qrc">
     <normaloff>:/Icons/images/update.png</normaloff>:/Icons/images/update.png</iconset>
   </property>
   <property name="text">
    <string>Check Updates</string>
   </property>
  </action>
  <action name="actionSet_Script_s_Folder">
   <property name="icon">
    <iconset resource="GCRes.qrc">
     <normaloff>:/Icons/images/Add folder.png</normaloff>:/Icons/images/Add folder.png</iconset>
   </property>
   <property name="text">
    <string>Set Script's Folder</string>
   </property>
   <property name="toolTip">
    <string>Adds user's script search path</string>
   </property>
  </action>
  <action name="actionWeb_Site">
   <property name="icon">
    <iconset resource="GCRes.qrc">
     <normaloff>:/Icons/images/Web.png</normaloff>:/Icons/images/Web.png</iconset>
   </property>
   <property name="text">
    <string>Web Site</string>
   </property>
   <property name="toolTip">
    <string>Go to Web site</string>
   </property>
  </action>
  <action name="actionEnable_Remote">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="GCRes.qrc">
     <normaloff>:/Icons/images/Network.png</normaloff>:/Icons/images/Network.png</iconset>
   </property>
   <property name="text">
    <string>Enable Remote</string>
   </property>
   <property name="toolTip">
    <string>Allow control from remote</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWindowTargeter</class>
   <extends>QWidget</extends>
   <header>qwindowtargeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GCRes.qrc"/>
 </resources>
 <connections/>
</ui>
